application/schemas.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str = None
```

Comment:
The `additional_info` field is set to `None` by default, which is not compatible with Pydantic's default behavior. Consider using `Optional[str]` for type hinting to indicate that this field is optional.